source/patterns/@aws-solutions-constructs/core/lib/elasticsearch-helper.ts

```diff
@@ -217,3 +237,122 @@ export function buildElasticSearchCWAlarms(scope: Construct): cloudwatch.Alarm[]
 
   return alarms;
 }
+
+function getSubnetIds(vpc?: ec2.IVpc, domainProps?: elasticsearch.CfnDomainProps):
```

Comment:
getSubnetIds says it is getting subnetIds.

A plural noun is a collection containing elements that are that noun, in this case string[]